inst/rmarkdown/templates/icesjms/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+# This template is largely functionally equivalent to that for elsevier
+# The two fields that are not in the elsevier format are:
+#      `corresponding_author` and `acknowledgements`
+title: Template for ICES Journal of Marine Science papers
+#date: "`r Sys.Date()`"
+author:
+  - name: Alice Anonymous
+    email: alice@example.com
+    corresponding_author: yes # Footnote using corresponding_author
+    affiliation:
+      - ABC
+      - DEF
+  - name: Bob Security
+    email: bob@example.com
+    affiliation: DEF
+    footnote: 1 # Footnote using reference
+  - name: Cat Memes
+    email: cat@anotheru.edu
+    affiliation: Another University
+    footnote:
+        - 1
+        - 2
+address:
+  - code: ABC
+    address: Address ABC
+  - code: DEF
+    address: Address DEF
+  - code: Another University
+    address: Address for Another University
+footnote:
+  - code: 1
+    text: "Equal contribution"
+  - code: 2
+    text: "Current email address: [cat@example.com](mailto:cat@example.com)"
+abstract: |
+  This is the abstract.
+
+  It consists of two paragraphs.
+acknowledgements: |
+  This is an acknowledgement.
+
+  It consists of two paragraphs.
+keywords:
+  - key
+  - dictionary
+  - word
+#fontsize: 12pt
+#spacing: halfline # could also be oneline
+#classoptions:
+#  - endnotes
+bibliography: mybibfile.bib
+csl: ices-journal-of-marine-science.csl
```

Comment:
Can this be downloaded from somewhere ? 

csl field can take a url and it would be better for us to use that instead of maintaining up to date such CSL file.